### ISSUE TITLE
[REQUEST] Add an about page

### ISSUE THREAD

**Issue by michele13** [igzivkov/LFS#21]
I've been watching your project with interest and I have a request to make

Could you add a page to the book where you 
- explain what are the reasons of the fork of the book? 
- Explain the choises that you make, for example porg, busybox, the non-standard root filesystem hierarchy? A lot of linux systems, especially hand-made ones, have some kind of philosophy on their back.

Also, I have a queation: what is your current relationship with the team of LFS and BLFS?

in short I'm really  curious about  the background of the book.

Thanks for your patience. You're doing a nice job

**Comment by igzivkov** [comment]
Sorry for late reply but I have been gone on holiday.

1) I don't consider this to be a fork of LFS books but rather my personal notebook as an easy way of keeping track of things that I do. If I come up with something that I consider really interesting to share with the community, maybe I'll setup a proper project or something. But right now, it's not that much different from LFS, just one guy's way of building his hobby OS.

2) All the choices that you noticed are just my experiments to see what works for me and what doesn't. For example, I really liked BusyBox in beginning but will switch back to coreutils & util-linux soon as I lost interest in keep fixing other packages to work with it. porg is the simplest way to keep track of packages but maybe I'll search for a real package manager in near future to automatically solve and build dependencies. I chose non-standard filesystem hierarchy really just because I'm too lazy to decide what should go to /usr etc.

I don't have any affiliation with LFS team currently except as a former contributor. I'm so glad that they are keeping up the great work as I can't imagine doing it all by myself or switching to any other distro.

How did you end up here? Was it to find out how to build more minimal LFS system or something else?

**Comment by michele13** [comment]
Il 07 gen 2017 8:47 PM, "Igor Živković" <notifications@github.com> ha
scritto:
>
> Sorry for late reply but I have been gone on holiday.
>
> I don't consider this to be a fork of LFS books but rather my personal
notebook as an easy way of keeping track of things that I do. If I come up
with something that I consider really interesting to share with the
community, maybe I'll setup a proper project or something. But right now,
it's not that much different from LFS, just one guy's way of building his
hobby OS.
>
> All the choices that you noticed are just my experiments to see what
works for me and what doesn't. For example, I really liked BusyBox in
beginning but will switch back to coreutils & util-linux soon as I lost
interest in keep fixing other packages to work with it. porg is the
simplest way to keep track of packages but maybe I'll search for a real
package manager in near future to automatically solve and build
dependencies. I chose non-standard filesystem hierarchy really just because
I'm too lazy to decide what should go to /usr etc.
>
> I don't have any affiliation with LFS team currently except as a former
contributor. I'm so glad that they are keeping up the great work as I can't
imagine doing it all by myself or switching to any other distro.
>
> How did you end up here? Was it to find out how to build more minimal LFS
system or something else?

I asked to the LFS community how to build an LFS distro using busybox
instead of all those packages, then I wanted to install XVesa to make a
sort of clone of TinyCore Linux (www.tinycorelinux.net) to undersrand how
it works and I was pointed to your interesting work.

**Comment by igzivkov** [comment]
Oh nice, I'm glad you found something useful here. If I may suggest projects to go further, they would be:
https://github.com/sabotage-linux/sabotage
http://suckless.org/
https://alpinelinux.org/
